File: 0001_0100/0052/0052.c
#include <stdio.h>

int main()
{
	FILE *f = fopen("input.txt", "r"), *q = fopen("output.txt", "w");
	char s[7];

	fscanf(f, "%s", s);
	fprintf(q, s[0] + s[1] + s[2] - s[3] - s[4] - s[5] ? "NO" : "YES");
	return 0;
}

File: 0001_0100/0052/0052.go
package main

import (
	"fmt"
	"os"
)

func main() {
	f, _ := os.Open("input.txt")
	q, _ := os.Create("output.txt")
	var s string
	fmt.Fscan(f, &s)
	r := "NO"
	if s[0]+s[1]+s[2]-s[3]-s[4]-s[5] == 0 {
		r = "YES"
	}
	fmt.Fprint(q, r)
}
